src/main/java/emu/grasscutter/command/commands/SpawnCommand.java

```diff
@@ -112,4 +186,34 @@ private Position GetRandomPositionInCircle(Position origin, double radius) {
         target.addX((float) (r * Math.cos(angle))).addZ((float) (r * Math.sin(angle)));
         return target;
     }
+
+    private static class SpawnParameters {
+        @Setter
```

Comment:
Stuff like this generally makes me prefer annotations on the same line (i.e. `@Setter public int id;`)

Reply:
the extra lines are mostly from intellijs autoformat, but I can change them back to one liners if wished.

Reply:
yes please!

My general view on it would be that class annotations should almost always be on a separate line, property/field annotations should be on the same line unless there's a crazy amount of them (which I'd probably set at 4+, since there's a few `@Transient @Getter @Setter private Foo foo`s in the codebase already and they look reasonable imo)